frontend/src/features/coverLetter/components/CoverLetterMenu.tsx

```
@@ -0,0 +1,53 @@
+const CoverLetterMenu = ({
+  documentId,
+  openReview,
+  isReviewOpen,
+}: {
+  documentId: number;
+  openReview: (value: boolean) => void;
+  isReviewOpen: boolean;
+}) => {
+  return (
+    <div className='w-48 rounded-md bg-white p-2 shadow-[0px_4px_20px_0px_rgba(41,41,41,0.12)]'>
+      <button className='flex h-11 w-full cursor-pointer items-center gap-2 rounded-xl px-3'>
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)